tests/spec/context_spec.lua

```
@@ -0,0 +1,296 @@
+        "end"
+      }
+    end
+    
+    local imports = context.extract_imports(1)
+    
+    assert.is_not_nil(imports, "Imports should not be nil")
+    assert.is_true(#imports >= 2, "Should find at least 2 imports")
+  end)
+  
+  it("should find context node for cursor position", function()
+    vim.bo[1] = { filetype = "lua" }
```

Luacheck (1.2.0)

[warning] mutating non-standard global variable 'vim'

(W112)